docs/content/doc/features/authentication.en-us.md

```diff
@@ -95,6 +95,18 @@ Both the LDAP via BindDN and the simple auth LDAP share the following fields:
     matching supplied login name against multiple attributes such as user
     identifier, email or even phone number.
   - Example: `(&(objectClass=Person)(|(uid=%[1]s)(mail=%[1]s)(mobile=%[1]s)))`
+- Enable user synchronization
+  - This option enables a periodic task that synchronizes the Gitea users with
+    the LDAP server. The default period is every 24 hours but that can be
+    changed in the
+    [app.ini](https://github.com/go-gitea/gitea/blob/master/custom/conf/app.ini)
```

Comment:
The path to the sample config file is what you have here, except it uses the .sample extension as well so that the sample config doesn't accidentally override the user's config file. So the full path is: https://github.com/go-gitea/gitea/blob/master/custom/conf/app.ini.sample